src/Microsoft.IdentityModel.Tokens.Saml/Saml2/Saml2SecurityTokenHandler.cs

```diff
@@ -64,11 +65,19 @@ public Saml2SecurityTokenHandler()
         /// <summary>
         /// Gets or set the <see cref="Saml2Serializer"/> that will be used to read and write a <see cref="Saml2SecurityToken"/>.
         /// </summary>
+        /// <exception cref="ArgumentNullException">'value' is null.</exception>
         public Saml2Serializer Serializer
         {
-            get;
-            set;
-        } = new Saml2Serializer();
+            get { return _serializer; }
+
```

Comment:
remove this line

Reply:
Done! Some properties in other classes put a space between the 'get' and 'set' so I wasn't sure what would be considered the correct style.